frontend/src/services/AuthorizationConst.ts

```diff
@@ -0,0 +1,14 @@
+
+import { IAuthService } from "./IAuthService";
+
+export const AuthorizationConst={
```

Comment:
ogólnie bedzie to dzialac tylko nie wiem czy ze wzgledu na projekt i cala architekture nie lepiej przerobic tego na klase, Wtedy tylko bedzie ciezkie odwolywanie sie do tokenu. Zostawiam do twojej decyzji moze zostac tak jak jest albo mozesz zmienic